### PR TITLE
WIP - Updating Editions Page

### PR DESCRIPTION
**Pull request by it33**
Includes switching to markdown from RST to make this page more accessible to non-technical reviewers

### CODE REVIEW

**Comment by cwarnermm** [comment]
@justinegeffen - Given the file format change from RST to Markdown, we'll need to ensure that landing pages and the LHS are also updated.

**Comment by cwarnermm** [comment]
@it33 - Is this PR ready for editorial review? May I remove the WIP from the PR title?

File: source/about/editions-and-offerings.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+- [Knowledge base and troubleshooting documentation](https://support.mattermost.com/hc/en-us) is available for common techincal issues. 
+- [Peer-to-Peer community support](https://mattermost.com/support/) via on-line forums. 
+
+Service entitlements from Mattermost, Inc: 
+- (Cloud Only) [SOC 2 Type 1 certification](https://mattermost.com/security/) with plans to add SOC 2 Type 2 in 2023. 
+
+
+### Mattermost Professional 
+
+Mattermost Professional is a paid subscription offering advanced configuration for software organizations from 25 to 100 users.
```
Comment (chenilim) [inline source/about/editions-and-offerings.md:98]: Recommend rewording or clarifying that "25 to 100 users" is just an ideal or guide.

**Comment by justinegeffen** [comment]
@it33, we're reviewing outstanding PRs and wanted to find out if this should be merged or kept aside? The only concern is the conflict which may impact the work you've done.

**Comment by it33** [comment]
Closing as I think @jasonblais has a more up-to-date version

**Pull request closed without merging**